azure-functions/acbs-function/helpers/retry.js

```diff
@@ -0,0 +1,6 @@
+const df = require('durable-functions');
```

Comment:
Any reason for the rename here? It feels like `retryOptions` is more specific to me ie if I read the below, I would assume that `retry` was the method to retry, not the `retryOptions`?
```
      const facilityBuyerTask = context.df.callActivityWithRetry('create-facility-guarantee', retry, {
        facilityIdentifier,
        acbsFacilityGuaranteeInput: acbsFacilityBuyerGuaranteeInput,
      });
```

Reply:
Agree I think `retryOptions` is clearer especially as it returns `new df.RetryOptions(...)`